### PR TITLE
[DCFS-17] Calculate Product Shipping Cost

### PR DESCRIPTION
**Pull request by kemysola**
## Task: `DOM MANIPULATION`
- `Changes you made to given task`
@adaugochi Good evening, Kindly review my pull Request.Thank you.

### CODE REVIEW

**Comment by Ireti203** [comment]
Kindly attach a screenshot of your product page showing the calculated product shipping cost

**Comment by Ireti203** [comment]
I also observed you have another opened PR yet to be merged. It is better you get an opened PR merged before opening another

**Comment by kemysola** [comment]
BD446B64-4A5A-44B2-887B-2C70A12E7CC7

Good morning @Ireti203  I have attached the screenshot. Kindly review ma

File: submissions/adekemiAwomolo/dom/product.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+quant = document.getElementById('product-quantity')
+btn = document.getElementById('order-btn');
+shipping = document.getElementById('shipping');
+total = document.getElementById('total');
+const Regex = /^[1-9]/;
+
+ function quantity_price(){
+     if(!quant.value.match(Regex)){
+         btn.disable = true;
+         btn.style.background = 'black';
```
Comment (adaugochi) [inline submissions/adekemiAwomolo/dom/product.js:11]: Instead of `black` use `grey`

File: submissions/adekemiAwomolo/dom/product.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+         quant.nextElementSibling.innerHTML ="  Kindly enter a valid Number";
+         quant.nextElementSibling.style.color = 'brown';
+        }
+        else{
+            btn.disable = false;
+            quant.nextElementSibling.innerHTML ="";
+            btn.style.background = 'orange';
+        }  
+        if(quant.value >= 1){
+           let productPrice = quant.value * 400;
```
Comment (adaugochi) [inline submissions/adekemiAwomolo/dom/product.js:21]: What does `400` represent?

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/adekemiAwomolo/dom/product.js (hunk 2 above)
Comment (kemysola) [reply submissions/adekemiAwomolo/dom/product.js:21]: The price of the product ma

Comment (adaugochi) [reply submissions/adekemiAwomolo/dom/product.js:21]: Also, assign it to a variable

Comment (kemysola) [reply submissions/adekemiAwomolo/dom/product.js:21]: Ok ma

Comment (kemysola) [reply submissions/adekemiAwomolo/dom/product.js:21]: But it’s assigned to a variable already ma.

**Review by adaugochi — APPROVED** [review]

**Pull request merged**